app/javascript/Components/__tests__/submission_selector.test.jsx

```diff
@@ -211,3 +220,42 @@ describe("SubmissionSelector", () => {
     expect(button).toBeDisabled();
   });
 });
+
+describe("SubmissionSelectorKeybinding", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    unbind_all_keybindings();
+  });
+
+  describe("bind keybindings", () => {
+    it("should have all expected keybindings", () => {
+      bind_keybindings();
```

Comment:
You can reduce the code duplication in the tests as follows:

1. Put the `bind_keybindings()` call into a `beforeEach` call
2. You can create parameterized tests to put each keybinding `expect` within a separate test case. See https://jestjs.io/docs/api#testeachtablename-fn-timeout; note that you can use `it.each` instead of `test.each` (`it` and `test` are aliases of each other in jest).